updater/spec/dependabot/integration_spec.rb

```diff
@@ -1,379 +0,0 @@
-        expect(log_message).to include("created", "dummy-pkg-b ( from 1.1.0 to 1.2.0 )")
-      end
-
-      end_to_end_job.run
-    end
-
-    it "instruments the package manager version" do
-      expect_any_instance_of(Dependabot::ApiClient).to receive(:record_package_manager_version)
-
-      end_to_end_job.run
-    end
-
-    context "when there is an exception that blocks PR creation" do
-      before do
-        allow(api_client).to receive(:create_pull_request).and_raise(StandardError, "oh no!")
-      end
-
-      it "notifies Dependabot API of the problem" do
-        expect(api_client).to receive(:record_update_job_error).
-          with(job_id, { error_type: "unknown_error", error_details: nil })
-
-        expect { end_to_end_job.run }.to output(/oh no!/).to_stdout_from_any_process
-      end
-
-      it "indicates there was an error in the summary" do
-        expect(Dependabot.logger).not_to receive(:info).with(/Changes to Dependabot Pull Requests/)
-        expect(Dependabot.logger).to receive(:info).with(/Dependabot encountered '1' error/)
-
-        expect { end_to_end_job.run }.to output(/oh no!/).to_stdout_from_any_process
-      end
-
-      it "does not raise an exception" do
-        expect { end_to_end_job.run }.to output(/oh no!/).to_stdout_from_any_process
-      end
-
-      context "when GITHUB_ACTIONS is set" do
-        before do
-          allow(Dependabot::Environment).to receive(:github_actions?) { "true" }
-        end
-
-        it "raises an exception" do
-          expect { end_to_end_job.run }.to raise_error(Dependabot::RunFailure).
-            and output(/oh no!/).to_stdout_from_any_process
-        end
-      end
-    end
-
-    context "when there is an exception that does not block PR creation" do
-      before do
-        # Pre-populate an error in the service
-        end_to_end_job.service.record_update_job_error(
-          job_id,
-          error_type: :epoch_error,
-          error_details: {
-            message: "What is fortran doing here?!"
-          }
-        )
-      end
-
-      it "indicates both the pr creation and error in the summary" do
-        expect(Dependabot.logger).to receive(:info).with(/Changes to Dependabot Pull Requests/) do |log_message|
-          expect(log_message).to include("created", "dummy-pkg-b ( from 1.1.0 to 1.2.0 )")
-          expect(log_message).to include("Dependabot encountered '1' error")
-        end
-
-        end_to_end_job.run
-      end
-
-      it "does not raise an exception" do
-        expect { end_to_end_job.run }.not_to raise_error
-      end
-
-      context "when GITHUB_ACTIONS is set" do
-        before do
-          allow(Dependabot::Environment).to receive(:github_actions?) { "true" }
-        end
-
-        it "raises an exception" do
-          expect { end_to_end_job.run }.to raise_error(Dependabot::RunFailure)
-        end
-      end
-    end
-  end
```

Comment:
Is this tested elsewhere? I don't think this is checked in the smoke test? If not it would be worth preserving somehow.